frontend-wasm/Cargo.toml

```diff
@@ -20,7 +20,7 @@ thiserror.workspace = true
 smallvec.workspace = true
 log.workspace = true
 anyhow.workspace = true
-wasmparser = "0.107"
```

Comment:
Any reason to keep this pinned to the build version vs minor? The former would've still installed 0.118.1 with a `cargo update` even though it mentions an older version. I don't have strong feelings either way, but I tend to pin to the minor version unless stability is a problem with a given crate.

Reply:
Minor would be enough here. I was unhappy with the breaking change below, so I was extra cautious.